src/pyxtp/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+find_package(Python 3 REQUIRED COMPONENTS Interpreter Development)
+find_package(pybind11 REQUIRED)
+
+file(GLOB_RECURSE BIND_SOURCES binds/*.cc)
+
+pybind11_add_module(xtp_binds ${BIND_SOURCES})
+target_include_directories(xtp_binds
+ PUBLIC
+ binds/include
+)
+
+target_compile_options(xtp_binds
+ PUBLIC
+)
+set_target_properties(xtp_binds
+  PROPERTIES
+  CXX_STANDARD 14
+  CXX_STANDARD_REQUIRED ON
+)
+
+
+target_link_libraries(xtp_binds
+ PUBLIC
+ VOTCA::votca_xtp
+)
+
+# # DPYXTP_VERSION_INFO is defined in the setup.py file
+target_compile_definitions(xtp_binds PRIVATE VERSION_INFO=${XTP_BINDS_VERSION_INFO})
```

Comment:
I looked at the auto-generated code and there is no `VERSION_INFO` define in there anywhere.

Reply:
I am removing that line